src/Elliptic/Systems/Poisson/Equations.cpp

```diff
@@ -51,25 +51,90 @@ void auxiliary_fluxes(
   }
 }
 
+/// \cond
+template <size_t Dim>
+void Fluxes<Dim, Geometry::FlatCartesian>::apply(
+    const gsl::not_null<tnsr::I<DataVector, Dim>*> flux_for_field,
+    const tnsr::i<DataVector, Dim>& field_gradient) noexcept {
+  flat_cartesian_fluxes(flux_for_field, field_gradient);
```

Comment:
[optional] If you wanted to,I think you could use `if constexpr` instead of partial specialization on the geometry. I'm fine with it either way :)

Reply:
I don't think I can, since the functions have different arguments, or am I missing something?

Reply:
yea, I didn't read carefully enough.... Sorry :(